### PR TITLE
Better Normalizers promoted from sandbox to full project

### PR DESCRIPTION
**Pull request by niccolox**
phing failed for me with sandboxed better_normalizers not found

changed this to url a full Drupal project

### CODE REVIEW

File: agov/drupal-org.make [hunk 1]
```diff
@@ -28,9 +28,8 @@ projects[panels][patch][] = https://www.drupal.org/files/issues/2699529-quote-co
 ; Development versions
 projects[better_normalizers][type] = module
 projects[better_normalizers][download][type] = git
-projects[better_normalizers][download][url] = https://git.drupal.org/sandbox/sam/2705067.git
+projects[better_normalizers][download][url] = https://git.drupal.org/project/better_normalizers.git
 projects[better_normalizers][download][branch] = 8.x-1.x
-projects[better_normalizers][download][revision] = 1cd9b4b50e91f5e595eb86813611dc368227c4b6
```
Comment (kimpepper) [inline agov/drupal-org.make]: Let's keep the revision reference.

Comment (niccolox) [inline agov/drupal-org.make]: no worries

**Comment by niccolox** [comment]
btw, using agov at http://gov.aegircloud.com

**Comment by kimpepper** [comment]
Made the changes in #809

**Pull request closed without merging**